### DISCUSSION TITLE
Build error with async branch during the update

### DISCUSSION THREAD

**Discussion by quadratmuede** [dani-garcia/vaultwarden#1571] [Q&A] [answered]
### Subject of the issue
Updating tot the latest version on async branch failed during the build.

### Your environment
Current Bitwarden_rs version: 1.19.0-01523fae (HEAD) (using the async branch)
Install method: manual build
Hosting platform: shared hosting on https://uberspace.de with about 1.5GB memory limit
Clients used:
Reverse proxy and version:
Version of mysql/postgresql:
Other relevant information: compile for sqlite

### Steps to reproduce
git pull origin async
cargo build -j 1 --release --features sqlite

### Expected behaviour
Build successful

### Actual behaviour
build error when compiling bitwarden_rs

### Relevant logs
```
   Compiling state v0.4.2 (https://github.com/SergioBenitez/state.git?rev=504ef71a#504ef71a)
error[E0658]: use of unstable library feature 'unsafe_cell_get_mut'
  --> /home/user/.cargo/git/checkouts/state-461be979d187f8b7/504ef71/src/shim.rs:41:40
   |
41 |             #[cfg(not(loom))] { self.0.get_mut() }
   |                                        ^^^^^^^
   |
   = note: see issue #76943 <https://github.com/rust-lang/rust/issues/76943> for more information

error: aborting due to previous error
```

**Comment by dani-garcia** [comment, marked as answer]
Your rust version might be a bit behind, that method has been stabilized in 1.50.0. Try to update the compiler if you can.

**Reply by quadratmuede** [reply]
Ah indeed I check the rust installed on the hosting platform and it is 1.49.0. Will contact them for a possible update. Thank you!
